shims/spark300/src/main/scala/com/nvidia/spark/rapids/shims/spark300/Spark300Shims.scala

```diff
@@ -119,6 +119,13 @@ class Spark300Shims extends SparkShims {
     }
   }
 
+  override def isGpuBroadcastNestedLoopJoin(plan: SparkPlan): Boolean = {
+    plan match {
+      case _: GpuBroadcastNestedLoopJoinExecBase => true
```

Comment:
sorry for my late comment I was looking at something else and realized this doesn't need to be in the shim, you are using the Base class here so there is nothing shim specific about it.